README.md

```diff
@@ -102,6 +174,98 @@ Reader.open('/usr/local/share/GeoIP/GeoIP2-Domain.mmdb').then(reader => {
 });
 ```
 
+## Database Exceptions
+
+If the database file does not exist, not readable, invalid, or there is a bug
+in the reader, the promise will be rejected with an `Error` with a message
+explaining the issue.
+
+If the the database file and the reader method does not match (e.g.
+`reader.city` is used with a Country database), a `BadMethodCalledError` will
+be raised.
+
+If the IP address is not found in the database, an `AddressNotFoundError` will
+be raised.
```

Comment:
I was trying to get this error with a bit of test code, but couldn't:

```
$ node test.js 192.168.0.1
aaaa Error: The address 192.168.0.1 is not in the database
```

Also there's an odd notion of what is a valid IP:

```
$ node test.js 1.1.1.1
1.1.1.1
AU
$ node test.js 8.987.654.321
8.987.654.321
SG
```